v3/package.json

```diff
@@ -83,95 +83,96 @@
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "@jsdevtools/coverage-istanbul-loader": "^3.0.5",
     "@svgr/webpack": "^6.5.1",
     "@talabes/json5-jest": "^2.0.0",
     "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
+    "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.4.3",
     "@types/d3": "^7.4.0",
-    "@types/jest": "^29.2.5",
+    "@types/jest": "^29.5.0",
     "@types/lodash": "^4.14.191",
     "@types/papaparse": "^5.3.7",
     "@types/pluralize": "^0.0.29",
-    "@types/react": "^18.0.26",
-    "@types/react-dom": "^18.0.10",
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
     "@types/rtree": "^1.4.28",
-    "@typescript-eslint/eslint-plugin": "^5.48.0",
-    "@typescript-eslint/parser": "^5.48.0",
+    "@typescript-eslint/eslint-plugin": "^5.55.0",
+    "@typescript-eslint/parser": "^5.55.0",
     "@welldone-software/why-did-you-render": "^7.0.1",
-    "autoprefixer": "^10.4.13",
-    "caniuse-lite": "^1.0.30001441",
+    "autoprefixer": "^10.4.14",
+    "caniuse-lite": "^1.0.30001467",
     "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^6.7.3",
     "cypress": "^11.2.0",
-    "eslint": "^8.31.0",
+    "eslint": "^8.36.0",
     "eslint-config-react": "^1.1.7",
-    "eslint-import-resolver-typescript": "^3.5.2",
+    "eslint-import-resolver-typescript": "^3.5.3",
     "eslint-plugin-chai-friendly": "^0.7.2",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^27.2.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-json": "^3.1.0",
-    "eslint-plugin-react": "^7.31.11",
+    "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-testing-library": "^5.9.1",
-    "eslint-webpack-plugin": "^3.2.0",
+    "eslint-plugin-testing-library": "^5.10.2",
+    "eslint-webpack-plugin": "^4.0.0",
     "file-loader": "^6.2.0",
+    "fs-extra": "^11.1.1",
     "html-webpack-plugin": "^5.5.0",
     "identity-obj-proxy": "^3.0.0",
     "istanbul-lib-coverage": "^3.2.0",
-    "jest": "^29.3.1",
+    "jest": "^29.5.0",
     "jest-canvas-mock": "^2.4.0",
-    "jest-environment-jsdom": "^29.3.1",
+    "jest-environment-jsdom": "^29.5.0",
     "json5-loader": "^4.0.1",
-    "mini-css-extract-plugin": "^2.7.2",
+    "mini-css-extract-plugin": "^2.7.5",
     "npm-run-all": "^4.1.5",
     "patch-package": "^6.5.1",
-    "postcss-loader": "^7.0.2",
+    "postcss-loader": "^7.1.0",
     "resize-observer-polyfill": "^1.5.1",
-    "sass": "^1.57.1",
+    "sass": "^1.59.3",
     "sass-loader": "^13.2.0",
     "script-loader": "^0.7.2",
-    "style-loader": "^3.3.1",
-    "ts-jest": "^29.0.3",
+    "style-loader": "^3.3.2",
+    "ts-jest": "^29.0.5",
     "ts-loader": "^9.4.2",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.4",
+    "typescript": "~4.9.5",
```

Comment:
Switching to `~` here as TypeScript doesn't following semantic versioning and so "minor" versions can contain breaking changes.